packages/grafana-data/src/types/config.ts

```diff
@@ -211,6 +211,7 @@ export interface GrafanaConfig {
   secretsManagerPluginEnabled: boolean;
   googleAnalyticsId: string | undefined;
   googleAnalytics4Id: string | undefined;
+  googleAnalytics4EnchancedTrackingIsActive: boolean;
```

Comment:
```suggestion
  googleAnalytics4EnchancedTrackingEnabled: boolean;
```

nit: just to follow the other naming conventions

Reply:
Good point, renamed it and fixed the typos https://github.com/grafana/grafana/pull/57619/commits/e0fdc96fb772af0dc143013ee3ad99f2d5bf5e7b